### DISCUSSION TITLE
ESlint error on Vercel build

### DISCUSSION THREAD

**Discussion by GrzegorzZajac000** [vercel/community#2825] [Bug Report]
### Page to Investigate

-

### Steps to Reproduce

package.json
![image](https://github.com/vercel/community/assets/78911177/4a595ae8-72e5-4741-8530-a4377152fcdd)

Errors from vercel's build:
![image](https://github.com/vercel/community/assets/78911177/a843d76b-1aa7-4b06-8d68-c31af499b54e)

Node: 18.x

Locally everything is fine, but on Vercel I've got these errors and I don't know why.

**Comment by smaeda-ks** [comment]
@GrzegorzZajac000 Hard to say without having a minimal reproducible example:
https://vercel.com/guides/creating-a-minimal-reproducible-example

Make sure your `yarn.lock` is up-to-date, and it's committed to your repository.

**Reply by GrzegorzZajac000** [reply]
Thank you. That was problem with `yarn.lock` file
